src/alire/alire-platforms.ads

```diff
@@ -76,6 +78,7 @@ private
    is (case Arch is
           when ARM64         => AARCH64,
           when Architectures => Arch,
+          when AMD64         => X86_64,
```

Comment:
You also have to add this conversion in the testsuite: https://github.com/alire-project/alire/blob/master/testsuite/drivers/helpers.py#L106

Reply:
Then I'd kindly request from @stcarrez to add a comment here pointing to the link given by Fabien.